### PR TITLE
Fixed feedback on 3/8 review

### PR DESCRIPTION
**Pull request by Ahmad-Ayyoub**

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -141,28 +140,28 @@ include::finish/src/main/java/io/openliberty/guides/system/SystemHealth.java[tag
 
 The `org.eclipse.microprofile` package contains the required MicroProfile Health Check classes.
 
-The `@Health` annotation indicates that this class is read to report the state of
-a service to the `http://localhost:9080/health` endpoint. The `@ApplicationScoped` annotation is
+The `@Health` annotation indicates that this class will be used to report the state of
```
Comment (evelinec) [inline README.adoc]: I think ID does not like future passive voice. "will be used to report" => "reports"

Comment (Ahmad-Ayyoub) [reply README.adoc]: Fixed

File: README.adoc [hunk 2]
```diff
@@ -72,15 +72,14 @@ mvn install liberty:start-server
...
+`http://localhost:9080/inventory/systems/localhost` URL retrieves the information for a specific host. The
+`http://localhost:9080/inventory/systems` URL retrieves the information for a list of all previously
 registered hosts.
 
-Point your browser to the MicroProfile Health Check endpoint at `http://localhost:9080/health`. From
-the endpoint, you can see the health of the two services as well as the overall health outcome of the
-application. One check shows the state of the `system`
-service, and the other shows the state of the `inventory` service. As you might expect,
-both services are in the `UP` state, and the overall outcome of the application is `UP`.
+Point to `http://localhost:9080/health` URL. From here, you can see the health of the two services as well as the overall
```
Comment (evelinec) [inline README.adoc]: => Next, go to `http://localhost:9080/health` URL to see the health of the two services as well as the..

File: README.adoc [hunk 3]
```diff
@@ -141,28 +140,28 @@ include::finish/src/main/java/io/openliberty/guides/system/SystemHealth.java[tag
...
 
 The `system` service self-checks. It reads the system property of the host to report a healthy state.
-If the `wlp.server.name` system property is
-`defaultServer`, the state is `UP`. Otherwise, the state is `DOWN`.
+If the `wlp.server.name` system property is `defaultServer`, the state is `UP`. Otherwise, the state is `DOWN`.
 
-Lastly, pass the `system` service name as an argument to the `named()`
-method to indicate the service that is reported.
+Lastly, the `named()` method from the interface needs to indicate the service to report the health, thus,
+passing the system service name as an argument.
```
Comment (evelinec) [inline README.adoc]: system to be backticked.

File: README.adoc [hunk 4]
```diff
@@ -141,28 +140,28 @@ include::finish/src/main/java/io/openliberty/guides/system/SystemHealth.java[tag
...
-a service to the `http://localhost:9080/health` endpoint. The `@ApplicationScoped` annotation is
+The `@Health` annotation indicates that this class will be used to report the state of
+`system` service to the `http://localhost:9080/health` URL. The `@ApplicationScoped` annotation is
 required because the health check service must stay active within the lifetime of the application
 to report the state.
 
-This class implements the `HealthCheck` interface. Use the interface to override the `call()`
-method.
+This class implements the `HealthCheck` interface from MicroProfile Health API. The `call()` method with `@Override`
+annotation from the interface is used to report the state of the service by returning a `HealthCheckResponse` object.
```
[the remark this answers is not in the document]
Comment (Ahmad-Ayyoub) [reply README.adoc]: Fixed

File: README.adoc [hunk 5]
```diff
@@ -198,14 +198,17 @@ Now, open the `inventory` service from the `src/main/java/io/openliberty/guides/
...
-The service invokes the `getPropertiesForHost()` method to display
-the system properties for a host name. The host name displays only if you set the
-`io.openliberty.guides.microprofile.inventory.inMaintenance` configuration to `false`.
-Otherwise, the service responds with a `Service is currently in maintenance.` message and
-a `503` response code. If the service displays the system properties, then
-the health of this service is `UP`. If the service is down for maintenance, its health is
-`DOWN'. Notice that the second condition in the `isHealthy()` method of the `InventoryHealth` class
-relies on the `getPropertiesForHost()` method.
+The `getPropertiesForHost()` method displays the system properties for a host name only if you set the
+`io.openliberty.guides.microprofile.inventory.inMaintenance` configuration to `false`. If this is the case,
```
[the remark this answers is not in the document]
Comment (Ahmad-Ayyoub) [reply README.adoc]: Fixed

File: README.adoc [hunk 6]
```diff
@@ -198,14 +198,17 @@ Now, open the `inventory` service from the `src/main/java/io/openliberty/guides/
...
+then the health of this service is `UP`.
+
+Otherwise, if the configuration is set to `true`, the service responds with a `ERROR: Service is currently in maintenance. Please contact: admin@guides.openliberty.io`
```
[the remark this answers is not in the document]
Comment (Ahmad-Ayyoub) [reply README.adoc]: Fixed

File: README.adoc [hunk 7]
```diff
@@ -198,14 +198,17 @@ Now, open the `inventory` service from the `src/main/java/io/openliberty/guides/
...
+message and a `503` response code. Thus, If the service is down for maintenance, its health is
```
[the remark this answers is not in the document]
Comment (Ahmad-Ayyoub) [reply README.adoc]: ERROR: Service is currently in maintenance. Please contact:   + inventoryConfig.getEmail().toString()) -> (admin@guides.openliberty.io)

[the remark this answers is not in the document]
Comment (Ahmad-Ayyoub) [reply README.adoc]: I did not want to leave Please contact: empty so I just added that e-mail

File: README.adoc [hunk 8]
```diff
@@ -198,14 +198,17 @@ Now, open the `inventory` service from the `src/main/java/io/openliberty/guides/
...
+`DOWN'.
+
+Notice that the `isHealthy()` method of the `InventoryHealth` class
+works with `getPropertiesForHost()` method through `InventoryConfig` object that modifies
+the same configuration file `/resources/CustomConfigSource.json`.
```
[the remark this answers is not in the document]
Comment (Ahmad-Ayyoub) [reply README.adoc]: Fixed

File: README.adoc [hunk 9]
```diff
@@ -259,21 +262,20 @@ Place the `@After` annotation on methods that execute after every test case. The
...
+* The `testIfServicesAreUp()` test case sends a request to the `http://localhost:9080/health` URL through the
+`connectToHealthEndpoint()` method from the `HealthTestUtil class`. This util method also verifies a `200` response code. If
+the request and verification are successful, then the util method returns the health result of the two services in
+`JsonArray` format. This result is asserted with the expected data when services are up through the `checkServicesStates()`
+private method. This private method uses the `getActualState()` method from the `HealthTestUtil` class to parse the actual
+state of each services in the `JsonArray` result."
+
+* The `testIfInventoryServiceIsDown()` method follows a procedure similar to the previous test case. The method first
+checks if the `services` states are `UP`. Next, the `io.openliberty.guides.microprofile.inventory.inMaintenance` property
+value is changed from `false` to `true` in the `/resources/CustomConfigSource.json` file. With the property update, the
```
[the remark this answers is not in the document]
Comment (Ahmad-Ayyoub) [reply README.adoc]: Fixed

File: README.adoc (hunk 2 above)
Comment (Ahmad-Ayyoub) [reply README.adoc]: Fixed

File: README.adoc (hunk 3 above)
Comment (Ahmad-Ayyoub) [reply README.adoc]: Fixed

File: README.adoc (hunk 7 above)
[the remark this answers is not in the document]
Comment (Ahmad-Ayyoub) [reply README.adoc]: @evelinec

[the remark this answers is not in the document]
Comment (evelinec) [reply README.adoc]: I don't think you need "Please contact:.." for this guide. That's the functionality for the config guide.

**Pull request merged**